nanshe_workflow.recipe/meta.yaml

```diff
@@ -30,7 +30,9 @@ requirements:
     - jupyter_contrib_nbextensions
     - dask-core
     - toolz >=0.7.3
+    - pyyaml
     - distributed
```

Comment:
So `pyyaml` should be required by `distributed` as it needs it for the configuration file. However it doesn't currently. Hence we include it here.